app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,35 @@
+.required-loan-types
+  label = t('loan_questions.required_loan_types')
+  .actions
+    a[data-expands='loan-types']
+      i.fa.fa-eye.fa-large
+      span = t('common.show')
+    a[data-hides='loan-types']
+      i.fa.fa-eye-slash
+      span = t('common.hide')
+
+  div data-expandable='loan-types'
+    - if @loan_question.parent
+      = f.collection_radio_buttons :override_associations,
+        [[false, t('loan_questions.inherit_parent')],
+        [true, t('loan_questions.override_parent')]], :first, :last,
+        item_wrapper_class: 'radio'
+
+    - hideLoanTypes = @loan_question.parent && !@loan_question.override_associations ? 'hidden': ''
+    p.loan-types-description = t('loan_questions.loan_types.description')
+    p.loan-types-description = t('loan_questions.loan_amounts.description')
+    .loan-types-container class="#{hideLoanTypes}"
+      .form-horizontal
+        - cfreqs = f.object.custom_field_requirements.sort_by { |i| i.loan_type.label.text }
+        = f.simple_fields_for :custom_field_requirements, cfreqs do |sf|
+          .form-group.row
+            .loan-type.col-sm-8
+              label
+                input.require-checkbox type="checkbox" checked=!sf.object.new_record?
+                = sf.input_field :_destroy, as: :hidden, value: sf.object.new_record?, class: 'destroy-field'
+                = sf.object.loan_type.label
+              = sf.input :option_id, as: :hidden
+            .amount.col-sm-4 class="#{sf.object.new_record? ? 'hidden-special' : ''}"
+              = sf.input_field :amount, step: '100', min: 0,
+                placeholder: t('loan_questions.loan_amounts.placeholder'),
+                class: "input-amount", value: sf.object.amount.to_i
```

Comment:
Simple form auto generates IDs/classes for fields. You shouldn't need to set one manually.

Also you should pretty much never need to manually set a value on a simple form field.